pkg/tasks/prerequisites.go

```diff
@@ -42,6 +42,26 @@ func installPrerequisites(s *state.State) error {
 	return s.RunTaskOnAllNodes(installPrerequisitesOnNode, state.RunParallel)
 }
 
+func kubeadmPreflightChecks(s *state.State) error {
+	s.Logger.Info("Running kubeadm preflight checks...")
+
+	return s.RunTaskOnControlPlane(
+		func(ctx *state.State, node *kubeoneapi.HostConfig, conn executor.Interface) error {
+			ctx.Logger.Info("	preflight...")
```

Comment:
```suggestion
			ctx.Logger.Info("preflight...")
```

Was this intentional for indentation?

Reply:
yep, it will looks like this

```
Running kubeadm preflight checks...
   preflight... node=xxx
   preflight... node=yyy
   preflight... node=zzz
```